src/alias/azext_alias/__init__.py

```diff
@@ -0,0 +1,60 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import timeit
+
+from knack.log import get_logger
+
+from azure.cli.core import AzCommandsLoader
+from azure.cli.core.commands import CliCommandType
+from azure.cli.core.commands.events import EVENT_INVOKER_PRE_CMD_TBL_TRUNCATE
+from azext_alias.alias import AliasManager
+from azext_alias._const import DEBUG_MSG_WITH_TIMING
+from azext_alias.telemetry import telemetry
```

Comment:
Move an many imports here to the actual methods that use them for perf. reasons.

I think only these ones would need to stay at the top.
```
from knack.log import get_logger
from azure.cli.core import AzCommandsLoader
```